src/main/java/stringcalculator/domain/UserInput.java

```diff
@@ -0,0 +1,29 @@
+package stringcalculator.domain;
+
+import java.util.Objects;
+import java.util.Scanner;
+
+public class UserInput {
+
+    String userInput;
+
+    public UserInput() {
+        userInput = userInputString();
+        if (isNullOrEmpty(userInput)) {
+            userInput = "0";
+        }
+    }
+
+    private String userInputString() {
+        Scanner scanner = new Scanner(System.in);
+        return scanner.nextLine();
+    }
```

Comment:
UserInput 객체가 사용자가 입력한 값을 상태로 가질 필요가 있을까요? 🤔

아래와 같이 변경해보면 어떨까요? 🤗 
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);

    private UserInput() {
    }

    public String userInputString() {
        String userInput = SCANNER.nextLine();
        if (isNullOrEmpty(userInput)) {
            userInput = "0";
        }
        return userInput;
    }
```